package.json

```diff
@@ -83,6 +83,7 @@
     "prettier": "^1.19.1",
     "prismjs": "^1.5.1",
     "react": "^16.12.0",
+    "react-pick-color": "^1.0.1",
     "react-dom": "^16.12.0",
```

Comment:
I don't see the `react-pick-color` being used in the code. I may miss something looking at the diff?